src/Base/StlFeatures.h

```diff
@@ -72,6 +73,10 @@ make_unique(std::size_t n)
 template<class T, class... Args>
 inline typename Detail::UniqueIf<T>::KnownBound
 make_unique(Args&&...) = delete;
+#else
+// Visual Studio 2015 implements make_unique
```

Comment:
What about older VS versions? This condition assumes that any VS supports make_unique.